index.html

```diff
@@ -4,9 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Portfolio skeleton</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" 
+  integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" 
+  crossorigin="anonymous">  
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio_Website</title>
 </head>
 <body>
-    <h1>Designing Portfolio Skeleton</h1>
+  <!--Header Section -->
+
+  <main class="head-section"> 
+      <nav class="top d-flex">
+        <h1>My Logo</h1>
+        <i class="fas fa-bars"></i>
+      </nav>     
```

Comment:
- [ ] Good job on implementing the previous request. However, you still have something to improve:
- Kindly put this `<nav>` tag inside a `<header>` tag.